config/go-app-ussd_public.cgg_UG.json

```diff
@@ -0,0 +1,402 @@
+    "{{pre}}Apr{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "{{pre}}Jun{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "{{pre}}Jul{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "{{pre}}Aug{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "{{pre}}Sep{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "{{pre}}Oct{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "{{pre}}Nov{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "{{pre}}Dec{{post}}": [
+        null,
+        "NOT IN SPREADSHEET"
+    ],
+    "You have an incomplete registration. Would you like to continue with this registration?": [
+        null,
+        "Okwehandikisa kwawe tikuhweire. Okayenzire kugumizamu nokwehandikisa oku?"
+    ],
+    "Welcome to FamilyConnect. Please choose your language": [
+        null,
+        "FamilyConnect nekwakira. Toorana orurimi"
+    ],
+    "Choose from:": [
+        null,
+        "Torana:"
+    ],
+    "Please enter the mobile number of the person who will receive the weekly messages. For example 0803304899": [
+        null,
+        "Taasamu enamba yesimu yomuntu oraagume kwakiira obutumwa bwa buri sande. Okyokureberaho 08003304899"
+    ],
+    "Choose:": [
+        null,
+        "Torana:"
+    ],
+    "You are already registered for baby messages.": [
+        null,
+        "Waherize kwehandikisa ahabwobutumwa bwo'mwaana."
+    ],
+    "Thank you. You will now receive messages about caring for the baby": [
+        null,
+        "Thank you. You will now receive messages about caring for the baby"
+    ],
+    "What language would you like to receive messages in?": [
+        null,
+        "Nirurimiki oruwokwenzire kutungamu obutumwa?"
+    ],
+    "Thank you. Your language preference has been updated and you will start to receive SMSs in this language.": [
+        null,
+        "Yebare, orurimi orworikwenda rwayerezibwa kandi nooza kutandika kutunga obutumwa omururimi rwaawe oru."
+    ],
+    "Please enter the new mobile number you would like to receive weekly messages on. For example 0803304899": [
+        null,
+        " Bambe taasamu enamba yesimu yomuntu oraguume kwakiira obutumwa bwa buri sande. Ekyokureberaho 0803304899"
+    ],
+    "Sorry, this number is already registered.": [
+        null,
+        "Enamba egi yakozesibwe"
+    ],
+    "Who will receive these messages?": [
+        null,
+        "Noha arayakire obutumwa obu"
+    ],
+    "Thank you. The number which receives messages has been updated.": [
+        null,
+        "Yebaare, enamba yokwakiira obutumwa yayerezibwa."
+    ],
+    "Why do you no longer want to receive messages?": [
+        null,
+        "Ahabwenki otakyayeenda kutunga obutumwa?"
+    ],
+    "We are sorry for your loss. Would you like to receive a small set of free messages from FamilyConnect that could help you in this difficult time?": [
```

Comment:
This hasn't yet been updated from the copy update here: https://github.com/praekelt/chms-jsbox/pull/56/files#diff-f11efc6ebde289613436f88de0401d97L1086